app/src/marketplace/styles/pcss/buttons.pcss

```diff
@@ -156,14 +155,19 @@
     letter-spacing: 1px;
 
     &:--enter {
-      box-shadow: 0 0 10px rgba(0, 0, 0, 0.15);
       color: var(--greyDark);
+      background-color: rgba(255, 255, 255, 1);
       text-decoration: none;
+      transform: scale(1.05);
```

Comment:
Not sure what's causing this but there's a little glitch at the end of the transition here (text moved down slightly). See

![Jun-03-2019 10-10-50](https://user-images.githubusercontent.com/320066/58787236-ad20e880-85e9-11e9-9473-f6087842e927.gif)

Also, there's a ticket for the "special" button – [`PLATFORM-552`](https://streamr.atlassian.net/browse/PLATFORM-552). @tumppi if you think it's done after changes in this PR please take the ticket over and mark as resolved.

I'd love to make the special button a separate component, or a theme for `button` and `a` tags, or in other words, avoid using global classnames (`btn-special` would be `Button.styles.special` or something).

Reply:
> Not sure what's causing this

Looks like some CPU/GPU floating point precision nonsense.

Was able to reproduce, adding `will-change: transform;` to `.btn.btn-special` fixes it:

![will-change-special-button](https://user-images.githubusercontent.com/43438/58858294-d15bf280-86d9-11e9-847f-a4710ac6bf29.gif)

Reply:
Good spot Mariusz and thanks Tim for finding a fix 👌